package.json

```diff
@@ -98,6 +98,6 @@
   "snyk": true,
   "dependencies": {
     "@uswds/compile": "1.2.0",
-    "@uswds/uswds": "3.8.2"
+    "@uswds/uswds": "github:uswds/uswds#al-alert-icon-alignment"
```

Comment:
**issue**: We should revert this change.

Reply:
Leaving it in for now while uswds/uswds#5636 is still in review. This way we can provide a demo of the on site.